### PR TITLE
Make verification script multichain

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-3610

### CODE REVIEW

File: packages/config/scripts/checkVerifiedContracts/addresses.test.ts
Comment (mateuszradomski) [inline packages/config/scripts/checkVerifiedContracts/addresses.test.ts:1]: Why not iterate over all known chainIds and test everything?

File: packages/config/scripts/checkVerifiedContracts/check.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  getOutputPath,
+  loadPreviouslyVerifiedContracts,
+  PROJECTS_OUTPUT_PATH,
+  saveResult,
+  VerificationMap,
+  VerificationMapPerChain,
+} from './output'
+import { verifyContracts } from './tasks'
+
+export const SUPPORTED_CHAINS = ['ethereum', 'arbitrum']
```
Comment (mateuszradomski) [inline packages/config/scripts/checkVerifiedContracts/check.ts:20]: I'd extract that to some configuration file higher up, it's kinda weird that this is here

File: packages/config/scripts/checkVerifiedContracts/output.ts [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
+  return `src/verification/${devId}/verified.json`
+}
 
 export async function loadPreviouslyVerifiedContracts(
-  filePath: string,
+  devId: string,
 ): Promise<Set<EthereumAddress>> {
+  const filePath = getOutputPath(devId)
+  if (!existsSync(filePath)) {
+    mkdirSync('src/verification/' + devId)
```
Comment (mateuszradomski) [inline packages/config/scripts/checkVerifiedContracts/output.ts:22]: It's better to compose that path in the following fashion:

```typescript
mkdirSync(path.dirname(getOutputPath(devId)))
```

File: packages/config/scripts/checkVerifiedContracts/output.ts [hunk 3]
```diff
@@ -28,23 +36,23 @@ export async function loadPreviouslyVerifiedContracts(
...
 ): Promise<FileStructure> {
-  if (!existsSync(filePath)) {
-    return { projects: {}, contracts: {} }
-  }
   const data = await readFile(filePath, 'utf-8')
   return FileStructure.parse(JSON.parse(data))
 }
 
-export async function saveResult(
+export async function saveResultPerChain(
```
Comment (mateuszradomski) [inline packages/config/scripts/checkVerifiedContracts/output.ts:43]: This function is not used anywhere

**Review by mateuszradomski — CHANGES_REQUESTED** [review]

File: packages/config/scripts/checkVerifiedContracts/addresses.test.ts
Comment (michalsidzej) [reply packages/config/scripts/checkVerifiedContracts/addresses.test.ts:1]: Good point

**Review by mateuszradomski — APPROVED** [review]

**Pull request merged**